src/main/java/org/folio/service/mapping/processor/translations/TranslationsHolder.java

```diff
@@ -73,6 +75,57 @@ public String apply(String updatedDate, int currentIndex, Translation translatio
       ZonedDateTime originDateTime = ZonedDateTime.parse(updatedDate, originFormatter);
       return targetFormatter.format(originDateTime);
     }
+  },
+
+  /**
+   * Forty character positions (00-39) that provide coded information about the record as a whole and about special
+   * bibliographic aspects of the item being cataloged.
+   * These coded data elements are potentially useful for retrieval and data management purposes.
+   * Format:
+   * 00-05 - Metadata.createdDate field in yymmdd format
+   * 06 is set to | (pipe character)
+   * 07-10 to publication[0] dateOfPublication if can be formatted else |||| (four pipe characters)
+   * 11-14 to publication[1] dateOfPublication if can be formatted else |||| (four pipe characters)
+   * 18-22 - each field set to |
+   * 23-29 - each field to be blank
+   * 30-34 - each field set to |
+   * 35-37 - if languages array is empty set it to "und",
+   * if one element, use it to populate the field (it should be 3 letter language code),
+   * if the array contains more than one language, then set it to "mul"
+   * 38-39 - each field set to |
+   */
+  SET_FIXED_LENGTH_DATA_ELEMENTS() {
+    private transient DateTimeFormatter originCreatedDateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+    private transient DateTimeFormatter targetCreatedDateFormatter = DateTimeFormatter.ofPattern("yyMMdd");
+    private String fieldPattern = "%s|%s%s||||||||       |||||%s||";
```

Comment:
This formatting doesn't look same when I parse the marc record:
<img width="921" alt="Screen Shot 2020-05-29 at 9 17 41 AM" src="https://user-images.githubusercontent.com/41596468/83263889-4b0faa80-a18d-11ea-97b2-bbfde97683dc.png">

Reply:
The spaces are not the same. I don't know if that's alright

Reply:
Good point, I will try to fix it

Reply:
It depends on the tool to parse records. MarcEditor shows all the 7 positions but replaces whitespaces by backslashes. I will describe it to Magda.

Reply:
![image](https://user-images.githubusercontent.com/40858461/83400228-26c10180-a40b-11ea-9911-a74bb534d44c.png)

Reply:
I am adding tests in my current PR to compare against a json instead of a marc record, it will help in these scenarios. You can leave it as is for now. Thanks!